app/forms/mappings/Formatters.scala

```diff
@@ -102,6 +102,33 @@ trait Formatters {
         baseFormatter.unbind(key, value.toString)
     }
 
+  private[mappings] def currencyFormatter(requiredKey: String, nonNumericKey: String, args: Seq[String] = Seq.empty): Formatter[BigDecimal] =
```

Comment:
should we create FormattersSpec and test currencyFormatter in isolation. I'm interested in more edge cases scenarios. I think right now we have just general controller tests. But I'm thinking of more cases just for this function, it should be easy testable.

Reply:
Pushed up now